src/System.Private.CoreLib/src/System/Enum.CoreCLR.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    partial class Enum
```

Comment:
CoreCLR prefers to duplicate everything, it should be `public abstract partial class Enum : ValueType, IComparable, IFormattable, IConvertible`

Reply:
We prefer to duplicate the modifiers (public, etc.). I do not think we generally duplicate the list interfaces unless it helps with readability.